app/models/daos/UserDAOImpl.scala

```diff
@@ -91,7 +91,7 @@ object UserDAOImpl {
    * Total number of contributing users
    */
   def countContributingUsers: Int = db.withTransaction { implicit session =>
-    val count = size - 1 + countAnonymousUsers + countTurkerUsers
+    val count = countRegisteredUsers + countAnonymousUsers + countTurkerUsers
```

Comment:
Is still double counting Turkers within countRegisteredUsers. Updating code to subtract countTurkersUsers.